### ISSUE TITLE
palvelurobotiikka tai palvelurobotit

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#7801]
#### Käsitteen tyyppi   

CONCEPT 

#### Ehdotettu termi suomeksi   

palvelurobotiikka tai palvelurobotit 

#### Aineisto jonka kuvailussa käsitettä tarvitaan (esim. nimeke tai URL)   

Robobisnes - havaintoja, kokemuksia ja näkymiä 

#### Perustelut ehdotukselle   

- 

**Ehdottajan organisaatio:** Kansalliskirjasto **Termiehdotus Fintossa:** [palvelurobotiikka tai palvelurobotit](http://finto.fi/yse/fi/page/Y507801)

**Comment by mariaforsen** [comment]
Sv. service/tjänsterobotik, service/tjänsterobotar

**Comment by mirmaid** [comment]
_Palvelurobotit_ kerran Melindassa.

**Comment by mirmaid** [comment]
Esim. https://www.jyu.fi/fi/ajankohtaista/arkisto/2018/07/palvelurobotit-yleistyvat-myos-suomessa. Toki määrittely voi olla haastavaa ja pelkkä _robotiikka_ & ala myös riittävä.

**Comment by YSO-kokous** [comment]
Jää termiehdotukseksi. Tarkkaillaan vakiintuuko. Nyt myös teollisuudessa palvelurobotteja (esim. varastorobotit).

**Comment by ilona-x** [comment]
Tätä olisi voinut käyttää väitöskirjan "[Consumer Responses to Service Robots - From Pre-Interaction to Post-Interaction](https://urn.fi/URN:ISBN:978-952-64-1407-2)" kuvailussa

**Comment by ilona-x** [comment]
YSOssa nyt palvelubotit KÄYTÄ chattibotit. Chattibotit ovat kuitenkin ilmeisesti vain yksi palvelurobottien laji. 

--
https://fi.wikipedia.org/wiki/Palvelurobotti
https://termipankki.fi/tepa/fi/haku/palvelurobotti

**Comment by mirmaid** [comment]
En ehkä edelleenkään olisi kauhean innostunut tästä, koska käsitteen määrittely tuntuu antavan aika paljon varaa tulkita suuri osa roboteista tähän ryhmään kuuluviksi. https://en.wikipedia.org/wiki/Robot#Service_robot

Voiko palvelubotti olla muutakin kuin chattibotti?

**Comment by ilona-x** [comment]
Joo, kyllähän noi palvelu**botit** on juurikin noita chattibotteja. Ja pärjään kyllä ilman näitä palvelurobotteja, ainakin toistaiseksi. 🧓 😄  Eli siis minun puolestani tämän voi jättää yhä odottamaan.

**Comment by YSO-kokous** [comment]
YSO-kokous ei nähnyt tarvetta käsitteelle toistaiseksi.
